docs/reference/glossary.md

```diff
@@ -32,6 +32,27 @@ An image that has no parent is a **base image**.
 [boot2docker](http://boot2docker.io/) is a lightweight Linux distribution made
 specifically to run Docker containers. The boot2docker management tool for Mac and Windows was deprecated and replaced by [`docker-machine`](#machine) which you can install with the Docker Toolbox.
 
+## bridge
+
+In terms of generic networking, a bridge is a Link Layer device which forwards
+traffic between networks. A bridge can be a hardware device or a software device
+running within a host machine's kernel.
```

Comment:
We could defer extra details to https://en.wikipedia.org/wiki/Bridging_(networking)